### ISSUE TITLE
Consider fields order

### ISSUE THREAD

**Issue by yoanmalie** [sebastiangreger/kirby3-commentions#116]
Hello, I try the last 2.0 beta 2 version and I got my fields in an order that I didn't choose.
Here my config:
```php
return [
  'sgkirby.commentions.commentfields' => [
    'name' => true,
    'email' => true,
    'website',
    'text',
  ],
];
```
Field are showed like:
- website
- name
- email
- text

The non-required input field "website" is always shown at first even if I try any other order in the config file.

I keep exploring this new version, thanks for all of this work!

**Comment by sebastiangreger** [comment]
Hi! Thank you for testing the beta and reporting these bugs :slightly_smiling_face:

Now that you point it out, this actually very obviously couldn't work to start with. As we mix both named (e.g. index `name`) and unnamed (i.e. index `0` for the "website" field), this inevitably is going to mess up the order as the array is rendered into fields:

```
Array
(
    [0] => website
    [name] => 1
    [email] => 1
    [1] => text
)
```

This was no problem in v1, as the frontend order was fixed in the form template, but now messes with the dynamic rendering as -- to my knowledge -- there is no way to determine the order these fields are originally defined in ...as soon as I read out the array it comes out "sorted" (correctly, as this is how PHP sees it) as above, not in the order it's written in the config file. That's also the reason why this only happens when both required and non-required fields are present.

Most likely this means the requirement to always give a value to each field; i.e. as follows:

```
return [
  'sgkirby.commentions.commentfields' => [
    'name' => true,
    'email' => true,
    'website' => false,
    'text' => false,
  ],
];
```

I'll investigate a bit further before making this (breaking) change and will include it in the next beta version.

**Comment by sebastiangreger** [comment]
This should be fixed with [v2.0-beta.3](https://github.com/sebastiangreger/kirby3-commentions/releases/tag/v2.0-beta.3), but indeed introduces a breaking change; see release notes.

New config syntax also [documented in the Readme](https://github.com/sebastiangreger/kirby3-commentions/blob/v2.0-beta.3/README.md#comment-fields).
